index.html

```diff
@@ -1429,6 +1429,35 @@ <h4>Rate obfuscation</h4>
           received from the platform collector during this penalty.
         </p>
       </section>
+      <section>
+        <h4>Rate obfuscation normative parameters</h4>
+        <p>
+          Base on implementation experience, implementers must use:
+          <ul>
+            <li>
+              a range in between 50 and 100 changes for PressureObserver's {{PressureObserver/[[MaxChangesThreshold]]}} internal slot.
+            </li>
+            <li>
+              a range in between 5000 milliseconds and 10000 milliseconds for PressureObserver's {{PressureObserver/[[PenaltyDuration]]}} internal slot.
+            </li>
+          </ul>
+          These values are subject to change and are updated based on further implementation experience and research findings.
```

Comment:
How about making this a Note?

Reply:
sure, I ll do that after @pes10k commenting.